manager/ui/hawtio/plugins/api-manager/ts/apimanPlugin.ts

```diff
@@ -230,7 +374,7 @@ module Apiman {
                 next();
             });
         });
-        
+
     }, true);
 
     hawtioPluginLoader.addModule(Apiman.pluginName);
```

Comment:
Is this one of the bits that's just staying in temporarily? Or did I accidentally bring that back in?